account-kit/react/src/components/auth/sections/OAuth.tsx

```diff
@@ -0,0 +1,48 @@
+import { useCallback } from "react";
+import { AppleIcon, FacebookIcon, GoogleIcon } from "../../../icons/oauth.js";
+import { assertNever } from "../../../utils.js";
+import { Button } from "../../button.js";
+import { useOAuthVerify } from "../hooks/useOAuthVerify.js";
+import type { AuthType } from "../types.js";
+
+type Props = Extract<AuthType, { type: "social" }>;
+
+// Not used externally
+// eslint-disable-next-line jsdoc/require-jsdoc
+export const OAuth = ({ ...config }: Props) => {
+  const { authenticate } = useOAuthVerify();
+  const startFlow = useCallback(() => authenticate(config), [authenticate]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useCallback has a missing dependency: 'config'. Either include it or remove the dependency array.
<!-- __reviewdog__:ChAxOGE3MjJjNGY2NzAwNGJhEgZyZGpzb24= -->